en/products/mozsocial/invite.ftl

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/mozsocial/invite/
+
+# HTML page title
+mozsocial-invite-join-the-waitlist = Join the waitlist for { -brand-name-mozilla-social }
+mozsocial-invite-welcome-to-mozillasocial = Welcome to { -brand-name-mozilla-social }!
+mozsocial-invite-please-join-our-waitlist = Please join our waitlist by completing the form below.
+mozsocial-invite-required-label = Required
+mozsocial-invite-first-name-label = First name
+mozsocial-invite-last-name-label = Last name
+mozsocial-invite-your-email-address-label = Your email address
+mozsocial-invite-country-label = Country
+
+# "Handle" is jargon for a user name.
+mozsocial-invite-twitter-handle-label = Twitter handle?
```

Comment:
Shall we change to `X (formerly Twitter)`?

Reply:
The Mozilla Social file was a smaller PR that came earlier but was not addressed and fixed when the bigger PR got merged with the older request.

Reply:
I will make a fix for issues that came up with the Mozilla Social PR now, thanks @peiying2